README.md

```diff
@@ -5,10 +5,27 @@ Tropical Example is a mock application for renting properties. It will be refere
 
 ## About EOSIO Labs <!-- omit in toc -->
 
-EOSIO Labs repositories are experimental.  Developers in the community are encouraged to use EOSIO Labs repositories as the basis for code and concepts to incorporate into their applications. Community members are also welcome to contribute and further develop these repositories. Since these repositories are not supported by Block.one, we may not provide responses to issue reports, pull requests, updates to functionality, or other requests from the community, and we encourage the community to take responsibility for these.
+EOSIO Labs repositories are experimental. Developers in the community are encouraged to use EOSIO Labs repositories as the basis for code and concepts to incorporate into their applications. Community members are also welcome to contribute and further develop these repositories. Since these repositories are not supported by Block.one, we may not provide responses to issue reports, pull requests, updates to functionality, or other requests from the community, and we encourage the community to take responsibility for these.
 
 ## Overview <!-- omit in toc -->
 
+### Try it out in Gitpod ###
+
+Gitpod [launches the app](https://gitpod.io/#https://github.com/EOSIO/tropical-example-web-app) for you. It starts the required blockchain in the background, launches the web server, and opens a preview window.
+NOTES: 
+ 1) There are several times during startup it might look like startup hangs, namely... near the end of the docker build, once the IDE comes up, and then once the preview shows.
+ 2) Sometimes when Gitpod launches the webapp preview, it does so prematurely.  Just click the small refresh circular arrow icon in the top left of the preview window.
+ 3) Gitpod generates a dynamic URL for the browser to access the app from. This URL is needed in numerous parts of the app, so note that there is code in this repo specifically meant for Gitpod compatibility. A comment has been added in those locations to point it out.
+ 4) To use Scatter in Gitpod, launch the demo in Gitpod, then go into Scatter's Networks section and add a custom network with the following settings as well as adding the account name. Note that these settings will need to be updated each time you launch the demo in Gitpod (because the URL will be different each time).
+ -Network Settings
```

Comment:
The formatting of `-Network Settings` and `-Adding account name` is a little wonky when the README is rendered:

<img width="876" alt="Screen Shot 2019-11-01 at 1 38 30 PM" src="https://user-images.githubusercontent.com/10996879/68043979-ecdd2080-fcac-11e9-8652-c19f0220e2e2.png">

Reply:
Might also help to bold `Host`, `Protocol` etc just to make everything easier to read